jobs.html

```diff
@@ -1,12 +1,46 @@
 ---
-title: QuTiP News
+title: QuTiP Jobs
 ---
 {% include header.html %}
 {% include navbar.html %}
 
 <div class="row">
 <div class="col-md-12">
 <h1 style="margin-bottom:20px">Job Announcements</h1>
+
+<h4>Technical Staff for QuTiP</h4>
+
+
+<p><strong>About this job</strong></p>
+<p>This is a job post for a [one]-year job opportunity to work on maintaining and improving QuTiP (the Quantum Toolbox in Python). Compensation is competitive with academic community standards and will depend on the candidate experience (PhD graduate or not, etc.). The candidate will work at RIKEN, Wako campus, Japan, within the Theoretical Quantum Physics Laboratory, RIKEN Cluster for Pioneering Research (Laboratory head: Franco Nori, Chief Scientist). RIKEN is Japan&rsquo;s basic science research institute. Wako is located in the Tokyo suburban area. You will coordinate with the QuTiP admin developers, under the supervision of Prof. Franco Nori.</p>
+<p><strong>Responsibilities</strong></p>
+<p>Coordinate with the QuTiP admin developers, under the supervision of Prof. Franco Nori, to maintain and improve QuTiP&rsquo;s features and functionalities, such as the ones already prioritized at <a href="https://github.com/qutip/qutip/wiki/Google-Summer-of-Code-2021">https://github.com/qutip/qutip/wiki/Google-Summer-of-Code-2021</a>.</p>
```

Comment:
@ajgpitch and I discussed moving the GSoC 2021 project list into qutip-doc. No need to change this now, just reminding us that we might need to update the reference at some point.